ts/features/itwallet/common/utils/itwClaimsUtils.ts

```diff
@@ -162,15 +162,7 @@ const DATE_FORMAT_REGEX = "^\\d{4}-\\d{2}-\\d{2}$";
 /**
  * Regex for the picture URL format which is used to validate the image claim as a base64 encoded png image.
  */
-const PICTURE_URL_REGEX = "^data:image\\/png;base64,";
-
-/**
- * Regex for the picture without URL format which is used to validate the image claim as a base64 encoded png image.
- * This is needed until the issuer adds the URL to the image claim.
- * TODO [SIW-1378]: remove this regex when the issuer adds the URL schema to the image claim.
- */
-const PICTURE_WITHOUT_URL_REGEX =
-  "(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)";
+const PICTURE_URL_REGEX = "^data:image\\/(png|jpg|jpeg|bmp|gif|webp);base64,";
```

Comment:
In the PR description you said `gif` and `webp` were not included?

Reply:
There was a commit I did not push, sorry.